crates/wysiwyg/src/dom/to_html.rs

```diff
@@ -25,11 +25,18 @@ where
         buf: &mut S,
         selection_writer: Option<&mut SelectionWriter>,
         state: ToHtmlState,
+        clean: bool,
     );
 
     fn to_html(&self) -> S {
         let mut buf = S::default();
-        self.fmt_html(&mut buf, None, ToHtmlState::default());
+        self.fmt_html(&mut buf, None, ToHtmlState::default(), true);
+        buf
+    }
+
+    fn to_internal_html(&self) -> S {
```

Comment:
My thinking was that `to_html` should output the final/best HTML and the new function should be more specialised. I'm not 100% on the naming of the new function...

Other options I considered:
- `to_ast()` / `to_ast_html()` (i.e. abstract syntax tree - though we'd then need to consider the output of `codeblock` which is split into `<pre>`/`<code>`)
- `to_editor_html()` (i.e. the HTML used by the editor UI?)

Reply:
Settled on `get_content_as_html()` and `get_content_as_message_html()`

So in both cases we're getting the editor content but in the special case we're getting it in a minimal 'message' format